### DISCUSSION TITLE
最好的防守就是进攻，大家有没有想过 除了SSR所有的方法可能都是GA人做的？

### DISCUSSION THREAD

**Discussion by sysalong** [XTLS/Xray-core#1386] [Ideas]
我在想有没有一种可能，ssr的作者被抓进去后，ga给了他另一个身份，让他带领团队创作了v2ray，xray等等（与其被动，不如主动），其实这些新的fq里面都有ga可识别（或者可解密）的方法，这样使用方法翻墙的人等于完全透明。大家仔细想想呢？

**Comment by kingwilliam** [comment]
你想多了，所有代码都是开源，如有你所说的，一早已抽了出来。

**Reply by sysalong** [reply]
开源不假，你有一行代码一行代码的去看么？  开源的程序哪个没出过漏洞，当然此漏洞非彼漏洞。

**Reply by reeceyng** [reply]
> 开源不假，你有一行代码一行代码的去看么？ 开源的程序哪个没出过漏洞，当然此漏洞非彼漏洞。

一行一行去看？*ray的代码早都已经被翻烂了

**Reply by sysalong** [reply]
呵呵，你说这话的意思就是，“你以为别人早都翻烂了”，指望别人发现问题告诉你？？？ 你问问你自己，你翻烂了么？

不再解释了。指望别人？？？我笑了兄弟。

**Reply by reeceyng** [reply]
> 呵呵，你说这话的意思就是，“你以为别人早都翻烂了”，指望别人发现问题告诉你？？？ 你问问你自己，你翻烂了么？
> 
> 不再解释了。指望别人？？？我笑了兄弟。

惭愧，我确实没翻烂。我只追踪着最近两年每次的commit。

另外提醒下你，你的博客泄露了你大量的信息，从qq号到真实照片不等，我看你也是从事网络安全的，希望注意一下

**Reply by sysalong** [reply]
谢谢提醒。能泄露的都是不在意的，毕竟也不是搞一天两天了。

**Comment by cross-hello** [comment]
You know recent years phone have by default trust government CA. So if 
trust system CA, man in the middle attack could be deployed without any 
detectable noise.

----------------------------------------

*Từ: *shrimp ***@***.***>
*Đến: *XTLS/Xray-core ***@***.***>
*CC: *Subscribed ***@***.***>
*Ngày: *Nov 28, 2022 07:16:36
*Chủ đề: *Re: [XTLS/Xray-core] 最好的防守就是进攻，大家有没有想过 除了SSR所有的方法可能都是GA人做的？ 
(Discussion #1386)

**Comment by ghost** [comment]
VPN 才是钓鱼的主战场，不论付费、免费
